.github/workflows/catalog-api-deploy.yml

```
@@ -0,0 +1,50 @@
+name: Deploy catalog-api
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+    - deploy
+  workflow_run:
+    workflows: ["catalog-api"]
+    branches: [dev]
+    types: [completed]
+
+env:
+  chart: catalog-api
+  namespace: eshop
+  app-name: eshop
+  chart-root: deploy/k8s/helm
+
+jobs:
+  DeployToK8s:
+    if: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'repository_dispatch' || github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
```

```suggestion
      - uses: actions/checkout@v2
```

Why are you using `v1`, that was released in Aug 2019 I believe.

---

💯Just copy/paste. The `build` workflows are using `@master` but I think pinning to a tag is better practice.